### ISSUE TITLE
SphereFace-64 and Baseline learning

### ISSUE THREAD

**Issue by melgor** [wy1iu/sphereface#27]
I have two question:
1. In weight initialization of released model you use sometimes `xavier`, sometimes `gaussion`. Any reason why? I have implemented this network in PyTorch and `xavier` for all layer works well for all network configuration expect `64` model (model diverge). And I'm searching reason why it does not work, maybe because of not mixed initialization? I checked other init method and non of them is working. 

2. I would like to learn baseline model using pure SoftMax. Did you add any layer between embedding (size 512) and classification layer (size num_class), like PReLu or Dropout? I have learned model (20) without adding anything, but it overfitt ( I split data to train/val set).

**Comment by wy1iu** [comment]
1. Gaussian is only used in residual blocks.
2. No, we do not use Dropout. Empirically, PReLU can indeed help convergence.

**Comment by melgor** [comment]
Thanks for the answer.
1. It look like it works. Do you have any reason why did you choose different init method for weight in your model? I did not meet such idea before. For all my model till SphereFace-64 one method of init just works fine (ex. MSRA or Xavier)? 
2. I will try to learn SphereFace with new init and see it this will improve situation. Currently SphereFace-36 overfitt less than SphereFace-20, what is uncommon.
